vanilla-framework/vanilla-framework-react
# Minor codebase cleanup

## Done

- Removed the following unused files/folders:
  - public/ folder (this is a component library so doesn't need one)
  - stories/ folder (stories live in individual component folders now)
  - .sass-lint.yml (at least for now, `vanilla-framework-react` contains no sass because `vanilla-framework` should be the source of truth for styling)
  - Various packages that weren't actually used anywhere. I'm sure there are a loooot more that can be cut (esp. after making distribution simpler)

## QA

- Check Percy for any visual diffs (should be none) - *Edit: weellll looks like placekitten changes the cat pictures every once in a while. Should probably remove that*
- Pull code
- Run `./run` and check that everything runs with no errors
- Run `npm pack` and check that the zip file package only contains the build folder, README and package.json
